### PR TITLE
feat: added categoryButtons

### PR DESCRIPTION
**Pull request by JasperVercammen**
Linking of de buttons to the next screen in next PR, 
I'm open for better name for `PillButton` 😄 

categories

### CODE REVIEW

File: src/_components/reward/style.ts [hunk 1]
```diff
@@ -41,11 +41,8 @@ export const RewardTileContainer = styled(TouchableRipple)`
   background-color: ${theme.palette.neutral[0]};
   border-radius: 8px;
   width: ${REWARD_TILE_WIDTH}px;
-  elevation: 1;
-  shadow-color: ${theme.palette.primary[800]};
-  shadow-offset: 0px 2px;
-  shadow-opacity: 0.05;
-  shadow-radius: 2px;
+
```
Comment (samvanhoeyicapps) [inline src/_components/reward/style.ts:44]: fan van enters?

File: src/shop/Shop.tsx [hunk 2]
```diff
@@ -2,13 +2,15 @@ import React from 'react';
...
 import { SafeAreaView } from '../_components';
+import { CategoryButtons } from './_components/categoryButtons/CategoryButtons';
 import { RewardsSection } from './_components/rewardsSection/RewardsSection';
 
 const Shop = () => {
   const { t } = useTranslation();
 
   return (
     <SafeAreaView isScrollable>
+      <CategoryButtons />
```
Comment (samvanhoeyicapps) [inline src/shop/Shop.tsx:13]: CategoryFilters maybe

File: src/shop/_components/categoryButtons/CategoryButtons.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+export const CategoryButtons = () => {
+  const { t } = useTranslation();
+
+  return (
+    <Styled.Container>
+      <FlatList
+        contentContainerStyle={{ paddingHorizontal: theme.common.defaultSpacing }}
+        data={CATEGORIES}
+        horizontal
+        keyExtractor={item => item.key}
```
Comment (samvanhoeyicapps) [inline src/shop/_components/categoryButtons/CategoryButtons.tsx:28]: why the key?

File: src/shop/_components/categoryButtons/CategoryButtons.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+import { TFilterRewardCategory, TFilterRewardSections } from '../../_queries/useGetRewards';
+import * as Styled from './style';
+
+type TCategoryListItem = { icon: TIconName; key: string; label: string; params: { category?: TFilterRewardCategory, type?: TFilterRewardSections } };
+const CATEGORIES: TCategoryListItem[] = [
+  { icon: 'Sport', key: '1', label: 'SHOP.SECTIONS.SPORT', params: { type: 'sport' } },
+  { icon: 'Activity', key: '2', label: 'SHOP.SECTIONS.DO', params: { category: 'Doen' } },
+  { icon: 'Charity', key: '3', label: 'SHOP.SECTIONS.CHARITY', params: { category: 'Goede doel' } },
+  { icon: 'Bag', key: '4', label: 'SHOP.SECTIONS.GADGET', params: { category: 'Gadget of item' } },
+  { icon: 'Clock', key: '5', label: 'SHOP.SECTIONS.LAST_CHANCE', params: { category: 'laatste kans' } },
```
Comment (samvanhoeyicapps) [inline src/shop/_components/categoryButtons/CategoryButtons.tsx:16]: doe je al iets met de params?

File: src/shop/_components/categoryButtons/CategoryButtons.tsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  const { t } = useTranslation();
+
+  return (
+    <Styled.Container>
+      <FlatList
+        contentContainerStyle={{ paddingHorizontal: theme.common.defaultSpacing }}
+        data={CATEGORIES}
+        horizontal
+        keyExtractor={item => item.key}
+        renderItem={({ item }) => <Styled.Button icon={item.icon} label={t(item.label)} onPress={() => { }} />}
```
Comment (samvanhoeyicapps) [inline src/shop/_components/categoryButtons/CategoryButtons.tsx:29]: Styled.PillButton

**Review by samvanhoeyicapps — DISMISSED** [review]

File: src/shop/_components/categoryButtons/CategoryButtons.tsx (hunk 3 above)
Comment (JasperVercammen) [reply src/shop/_components/categoryButtons/CategoryButtons.tsx:28]: best practice

File: src/shop/_components/categoryButtons/CategoryButtons.tsx (hunk 4 above)
Comment (JasperVercammen) [reply src/shop/_components/categoryButtons/CategoryButtons.tsx:16]: In volgende PR

**Review by samvanhoeyicapps — APPROVED** [review]

**Pull request merged**